sapp/sarif.py

```diff
@@ -42,6 +55,7 @@ class SARIF:
     _tool_warning_code_ranges: Dict[str, Tuple[int, int]] = field(default_factory=dict)
     driver: Dict[str, Union[str, List[Dict[str, str]]]] = field(default_factory=dict)
     results: List[SARIFResult] = field(default_factory=list)
+    tool: str = "pysa"
```

Comment:
moved the tool to a field since it is used internally and not worth it passing it around different functions